robocop/checkers/tags.py

```diff
@@ -35,6 +38,22 @@ def visit_DefaultTags(self, node):  # noqa
     def visit_Tags(self, node):  # noqa
         self.check_tags(node)
 
+    def visit_Documentation(self, node):  # noqa
+        if self.is_keyword:
+            *_, last_line = node.lines
+            filtered_line = filter(lambda tag: tag.type not in Token.NON_DATA_TOKENS and tag.type != Token.DOCUMENTATION, last_line)
```

Comment:
What about just using list comprehension, any pros of using filter?
```
filtered_line = [token for token in last_line if token not in Token.NON_DATA_TOKENS and token.type != Token.Documentation]
```

Reply:
Using list comprohension is more pythonic, but using `filter` may be more faster.